cypress/e2e/marc-authority/marc-authority-edit.cy.js

```diff
@@ -280,4 +280,20 @@ describe('MARC Authority -> Edit Authority record', () => {
     QuickMarcEditor.constinueWithSaveAndCheck();
     QuickMarcEditor.checkFieldAbsense('382');
   });
+
+  it('C375172 Save "MARC authority" record with deleted field and updated fields (spitfire)', { tags: [TestTypes.criticalPath, DevTeams.spitfire] }, () => {
+    MarcAuthorities.searchBy(testData.authority.searchOption, testData.authority.title);
+    MarcAuthorities.selectTitle(testData.authority.title);
+    MarcAuthority.edit();
+    // Waiter needed for the whole page to be loaded.
```

Comment:
can we wait for special string, element that is loaded?

Reply:
need to wait until all api requests are processed, which are not displayed on the UI. The problem is that it is not yet clear which of them exactly should be expected.